applications/luci-app-aria2/luasrc/view/aria2/web_script.htm

```diff
@@ -0,0 +1,77 @@
+<script type="text/javascript">//<![CDATA[
+XHR.poll(5, '<%=url([[admin]], [[services]], [[aria2]], [[status]])%>', null,
+	function(x, data) {
+		var tb = document.getElementById('aria2_status');
+		if (data && tb) {
+			if (data.running) {
+				var links = '<em><%:The Aria2 service is running.%></em>';
+				if (data.yaaw) {
+					links += '<input class="cbi-button mar-10" type="button" value="<%:Open YAAW%>" onclick="openWebUI(\'yaaw\');" />';
+				}
+				if (data.webui) {
+					links += '<input class="cbi-button mar-10" type="button" value="<%:Open WebUI-Aria2%>" onclick="openWebUI(\'webui-aria2\');" />';
```

Comment:
And there shows the button to yaaw or webui-aria2.
If not installed either of them, it's OK.
So ```+!PACKAGE_webui-aria2:yaaw``` can be removed